taxcalc/current_law_policy.json

```diff
@@ -3093,6 +3093,50 @@
         "out_of_range_action": "stop"
     },
 
+    "_PT_exclusion_rt": {
+        "long_name": "Pass-through income exclusion rate",
+        "description": "Fraction of qualified business income excluded from taxable income.",
+        "section_1": "Personal Income",
+        "section_2": "Pass-Through",
+        "irs_ref": "",
+        "notes": "Applies to e00900 + e26270",
+        "row_var": "FLPDYR",
+        "row_label": ["2013"],
+        "start_year": 2013,
+        "cpi_inflated": false,
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [0.0],
+        "range": {"min": 0.0, "max": 1.0},
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop"
+    },
+
+    "_PT_exclusion_limit": {
+        "long_name": "Limit on pass-through income exclusion",
+        "description": "If taxpayer has partnership/ S corporation income, the amount of business income excluded from taxable income may not exceed this fraction of wages.",
+        "section_1": "Personal Income",
+        "section_2": "Pass-Through",
+        "irs_ref": "",
+        "notes": "",
+        "row_var": "FLPDYR",
+        "row_label": ["2013"],
+        "start_year": 2013,
+        "cpi_inflated": false,
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [9e99],
+        "range": {"min": 0.0, "max": 9e99},
```

Comment:
I'm a little confused about this parameter. The description says it's a fraction so shouldn't the value range be 0.0-1.0?

Reply:
I think it's ok for this to be greater than 1. For example, you could limit PT income to 200% of wages.

Reply:
When applied in the TCJA, it should be applied as a fraction. However, in the default state, it shouldn't apply at all. Thus the default value of 9e99, which makes the restriction inoperative unless deliberately changed.

Reply:
Ah gotcha. Thanks for the clarification.